src/host/output.cpp

```diff
@@ -512,8 +512,6 @@ void CloseConsoleProcessState()
 
     HandleCtrlEvent(CTRL_CLOSE_EVENT);
 
-    gci.ShutdownMidiAudio();

```

Comment:
FYI reading another PR, I just realized that this change isn't quite correct. This should still call `midiAudio.BeginSkip()`, just like `ControlCore` does on shutdown.